src/apps/Permissions/ManageRolePanel.js

```diff
@@ -356,17 +377,47 @@ class ManageRolePanel extends React.PureComponent {
             </Button>
           )}
 
-          {(this.state.newRoleManagerValue === ANY_ENTITY ||
-            this.state.assignEntityAddress === ANY_ENTITY) && (
+          {(granteeIsAnyEntity || managerIsAnyEntity) && (
+            <Info
+              mode="warning"
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+            >
+              Be aware that granting the {granteeIsAnyEntity && 'permission'}{' '}
+              {bothIsAnyEntity && 'or'} {managerIsAnyEntity && 'manager role'}{' '}
+              to this address will let anyone{' '}
+              {granteeIsAnyEntity && 'perform this action'}{' '}
+              {bothIsAnyEntity && 'or'}{' '}
+              {managerIsAnyEntity && 'manage this permission.'}
+            </Info>
+          )}
+
+          {(granteeIsBurnEntity || managerIsBurnEntity) && (
+            <Info
+              mode="warning"
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+            >
+              Be aware that granting the {granteeIsBurnEntity && 'permission'}{' '}
+              {bothIsBurnEntity && 'or'} {managerIsBurnEntity && 'manager role'}{' '}
+              to this address will "burn" the permission, effectively freezing
+              it and disallowing any further changes.
+            </Info>
+          )}
+
+          {managerIsUnassignedEntity && (
             <Info
               mode="warning"
               css={`
                 margin-top: ${3 * GU}px;
               `}
             >
-              Be aware that assigning the permission or manager role to this
-              address will let anyone perform this action or manage this
-              permission.
+              Be aware that granting the manager role to this address will
+              remove the currently assigned one. The permission can only be
+              granted or revoked if it is initialized again (requiring the
+              “Create permission” action on the ACL app).
```

Comment:
I more or less mimicked the copy from the "Remove the manager" option. We might want to think more about this copy though.